static/app/types/group.tsx

```diff
@@ -849,6 +849,7 @@ export interface GroupOpenPeriod {
   end: string;
   isOpen: boolean;
   start: string;
+  lastChecked?: string;
```

Comment:
should this be optional? if not, maybe `string | null` would be a better match for the backend type

Reply:
updating the backend PR - it shouldn't ever be null in theory. I'm marking it optional for now while we add the field and will change to required after the PRs land.